dev/constraints.h

```diff
@@ -75,6 +75,70 @@ namespace sqlite_orm {
             }
         };
 
+        struct generated_always_string {
+            operator std::string() const {
+                return "GENERATED ALWAYS";
```

Comment:
this struct is used at only one place so it can be removed and its code can be moved to the place of usage